src/common/reducers/snaps.js

```diff
@@ -10,7 +10,9 @@ export function snaps(state = {
     case ActionTypes.FETCH_SNAPS:
       return {
         ...state,
-        isFetching: true
+        isFetching: true,
+        success: false,
+        error: null
```

Comment:
I wonder if we don't want to clear `snaps` too (if we clear the success flag).

But on the other hand, we probably don't want to clear data that we already have, so maybe it's not an issue.